README.md

```diff
@@ -16,17 +16,17 @@ This repo currently powers three separate forms:
 You will either need a copy of `meshdb` running locally, or point your local copy of this package to the beta/production
 endpoints in order to do development:
 
-- To setup against beta/prod, continue to the next section and use `https://db.nycmesh.net` as the value for `NEXT_PUBLIC_MESHDB_URL`
+- To setup against beta/prod, continue to the next section and use `https://devdb.nycmesh.net` as the value for `MESHDB_URL`
 - To setup a local copy of meshdb, follow the instructions in the readme [there](https://github.com/nycmeshnet/meshdb) first.  
-   Then continue to the next section below and use the value `http://127.0.0.1:8000` for `NEXT_PUBLIC_MESHDB_URL`
+   Then continue to the next section below and use the value `http://127.0.0.1:8000` for `MESHDB_URL`
 
 To get started with this package, first, create your own copy of the `.env` file:
 
 ```
 cp .env.sample .env
 ```
 
-Edit the `NEXT_PUBLIC_MESHDB_URL` based on your choice above in your favorite text editor. For example:
+Edit the `MESHDB_URL` and `NEXT_PUBLIC_MESHDB_URL` values based on your choice above in your favorite text editor. For example:
```

Comment:
@WillNilges why are there two separate variables now?

Reply:
I think `NEXT_PUBLIC_MESHDB_URL` is vestigal and can be removed. NEXT_PUBLIC variables get compiled into the image so they can be accessed client side, which we can't do if we want to have a dev and prod enviornment.

I think you can just remove it, but grep through to make sure. `MESHDB_URL` is the thing to use.